app/build.gradle

```diff
@@ -17,6 +17,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    useLibrary 'org.apache.http.legacy'
```

Comment:
Legacy http client is legacy; but I'll probably just replace it with another version of HTTPConnect class when other branches will be to be merged